Interview Questions

```diff
@@ -0,0 +1,78 @@
+# Interview questions for Junior Front-End Web Developers
+
+The purpose of this article is to familiarize fresh Front-End Web Developers with text book questions. If you are just getting into the developer world there is a good chance you will feel nervous when interviewing for a developer's positon. Because of this, some relatively basic questions could feel more difficult then how they actually are. A good counter for this scenario is to be properly prepared. Without further ado, here are a few questions that you could encounter:
+
+1)	Can you explain the meaning of “inline-block”?  
```

Comment:
> In what context?

would be my reply.

I think you should mention that it is in the context of CSS styling. It would be even better if you were to ask what does `display-style: inline-block` stand for. 

Front-end is a vast area; and when you say `inline-block` out of the blue, I would think that interviewer is probably asking for `inline-block` code scoping in JS. And then I would be more confused because I have never heard of anything like that.